### PR TITLE
Added the nearby airports page

### PR DESCRIPTION
**Pull request by algjohnston**
Team 2!

The nearby airports page has been added to the tabs and successfully displays. We all worked on it from 5:30 to now!

### CODE REVIEW

File: Model/NearbyAirport.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+/// <summary>
+/// Written by Hiba Seraj
+/// Represents nearby airports.
+/// </summary>
+/// <param name="id"></param>
+/// <param name="city"></param>
+/// <param name="miles"></param>
+/// <param name="isVisited"></param>
+public class NearbyAirport(string id, string city, int miles, bool isVisited)
```
[the remark this answers is not in the document]
Comment (algjohnston) [reply Model/NearbyAirport.cs:11]: What do you mean by adding the NearbyAirports to a collection?

[the remark this answers is not in the document]
Comment (rogersm-uwosh) [reply Model/NearbyAirport.cs:11]: Just use a ObservableCollection<Airport>NearbyAirports { get; set;}

[the remark this answers is not in the document]
Comment (algjohnston) [reply Model/NearbyAirport.cs:11]: What about miles?

File: NearbyAirports/AirportVisitedConvertor.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
+﻿using System.Globalization;
+
+namespace Lab2_Solution.NearbyAirports;
+
+/// <summary>
+/// Alexander Johnston wrote this class
+///
+/// Converts boolean to a red circle or green circle image source.
+/// </summary>
+public class AirportVisitedConvertor : IValueConverter
```
[the remark this answers is not in the document]
Comment (algjohnston) [reply NearbyAirports/AirportVisitedConvertor.cs]: Fixed 🥇

File: NearbyAirports/NearbyAirportsPage.xaml [hunk 3]
```diff
@@ -0,0 +1,76 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<!-- James Thao (Provided Grid Layout)
+     Jason Wang(Provided item template)
+     Alexander Johnston (Provided the image in the item template)
+-->
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
```
[the remark this answers is not in the document]
Comment (algjohnston) [reply NearbyAirports/NearbyAirportsPage.xaml:7]: Fixed 🥇

File: Model/NearbyAirport.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (algjohnston) [reply Model/NearbyAirport.cs]: We removed the NearbyAirports class

**Comment by rogersm-uwosh** [comment]
There's a minor question of tab bar title, but otherwise this was fine. Thanks, team, for your contribution!

**Pull request merged**